taco/internal/api/internal.go

```diff
@@ -0,0 +1,210 @@
+package api
+
+import (
+	"log"
+	"net/http"
+	"os"
+
+	"github.com/diggerhq/digger/opentaco/internal/domain"
+	"github.com/diggerhq/digger/opentaco/internal/middleware"
+	"github.com/diggerhq/digger/opentaco/internal/rbac"
+	"github.com/diggerhq/digger/opentaco/internal/repositories"
+	unithandlers "github.com/diggerhq/digger/opentaco/internal/unit"
+	"github.com/labstack/echo/v4"
+)
+
+
+func RegisterInternalRoutes(e *echo.Echo, deps Dependencies) {
+	webhookSecret := os.Getenv("OPENTACO_WEBHOOK_SECRET")
```

Comment:
I think we should make it more explicit like a flag called OPENTACO_ENABLE_INTERAL_ENDPOINTS since the flag OPENTACO_WEBHOOK_SECRET doesn't really tell me that it will lead to alot of admin endpoints are going to be exposed